js-miniapp-sample/src/pages/window_actions.js

```diff
@@ -0,0 +1,51 @@
+// @flow
+import React from 'react';
+
+import { Button, Card, CardContent, makeStyles } from '@material-ui/core';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    height: 300,
+    width: '100%',
+  },
+  content: {
+    height: '50%',
+    justifyContent: 'space-between',
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+    fontSize: 18,
+    color: theme.color.primary,
+    fontWeight: 'bold',
+  },
+}));
+
+const WindowActions = () => {
+  const classes = useStyles();
+  const onAlert = () => {
+    window.alert('This is window alert!');
+  };
+  const onConfirm = () => {
+    window.confirm('This is window confirm!');
+  };
+  const onPrompt = () => {
+    window.prompt('This is window prompt!');
```

Comment:
Can we also print the string/message entered in the prompt dialog? 
Because in iOS JS dialogs are not supported by default, we intercept and display native iOS dialogs and just want to make sure the value is returned.